db/migration/20240122223340_fix_null_image_files.go

```diff
@@ -0,0 +1,509 @@
+package migrations
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/pressly/goose/v3"
+)
+
+func init() {
+	goose.AddMigrationContext(Up20240122223340, Down20240122223340)
+}
+
+func Up20240122223340(ctx context.Context, tx *sql.Tx) error {
+	_, err := tx.ExecContext(ctx, `
+drop index if exists album_full_text;
+drop index if exists album_alphabetical_by_artist;
+drop index if exists album_order_album_name;
+drop index if exists album_order_album_artist_name;
+
+drop index if exists artist_full_text;
+drop index if exists artist_order_artist_name;
+
+drop index if exists media_file_full_text;
+drop index if exists media_file_order_album_name;
+drop index if exists media_file_order_artist_name;
+drop index if exists media_file_order_title;
+drop index if exists media_file_bpm;
+drop index if exists media_file_channels;
+
+alter table album
+    add image_files_new varchar not null default '';
+update album
+set image_files_new = image_files
+where image_files is not null;
+alter table album
+    drop image_files;
+alter table album
+    rename image_files_new to image_files;
+
+alter table album
+    add full_text_new varchar not null default '';
```

Comment:
All full_text columns already have a default, so you don't need to do it again, even though the type is `varchar(255)` and not `varchar`